api/models/likeDao.js

```diff
@@ -0,0 +1,41 @@
+const { AppDataSource } = require('./data_source');
+
+const getLikeUserId = async (userId, productId) => {
+    const result = await AppDataSource.query(
+        `
+		SELECT 
+           likes.user_id AS userId,
+           likes.product_id AS prodcutId
+		FROM likes
+		WHERE user_id =? AND product_id = ?`,
+        [userId, productId]
+    );
+    return result[0];
+};
+
+const createLike = async (userId, productId) => {
+    await AppDataSource.query(
```

Comment:
```suggestion
    await appDataSource.query(
```
대문자로 시작하는 것은 주로 클래스 그 자체에만 사용하는것이 컨벤션 입니다. 이로인해 만들어진 인스턴스에는 대문자가 아닌 `appDataSource`와 같은 이름을 갖도록 수정해주세요 .